### PR TITLE
Add Deploy Scripts to dss-bridge

### PR DESCRIPTION
**Pull request by hexonaut**
Adding scripts to deploy dss-bridge and xdomain-dss.

### CODE REVIEW

File: packages/dss-bridge/script/lib/XDomainDss.sol [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+// Tools for deploying and setting up an xdomain-dss instance
+library XDomainDss {
+
+    function switchOwner(address base, address newOwner) internal {
+        AuthLike(base).rely(newOwner);
+        AuthLike(base).deny(address(this));
+    }
+
+    function deploy(address pauseProxy) internal returns (DssInstance memory dss) {
```
Comment (gbalabasquer) [inline packages/dss-bridge/script/lib/XDomainDss.sol:39]: Is it really the pauseProxy or should be the gov relay guest?

Comment (hexonaut) [reply packages/dss-bridge/script/lib/XDomainDss.sol:39]: Yes was going to change this.

File: packages/dss-bridge/script/DeployExistingTokenBridge.s.sol [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.14;
+
+import "forge-std/Script.sol";
+
+import "./lib/XDomainDss.sol";
+
+// To deploy on a domain with an existing DAI + Token Bridge
```
[the remark this answers is not in the document]
Comment (hexonaut) [reply packages/dss-bridge/script/DeployExistingTokenBridge.s.sol:9]: Yes this is a draft. I'm not finished. :)

**Comment by hexonaut** [comment]
Moved to https://github.com/makerdao/dss-bridge/pull/2

**Pull request closed without merging**